lucille-core/src/main/java/com/kmwllc/lucille/connector/jdbc/DatabaseConnector.java

```diff
@@ -149,6 +151,13 @@ public void execute(Publisher publisher) throws ConnectorException {
         // log.info("Add Field {} Value {} -- Doc {}", columns[i-1].toLowerCase(), rs.getString(i), doc);
         String fieldValue = rs.getString(i);
         if (fieldValue != null) {
+
+          // TODO THROWS ERROR IF IS SET AND THE RESULT HAS A COLUMN WITH THE SAME NAME
+          // todo also consider removing the field with the original name to avoid duplication
+          if (fieldName.equals("id")) {
+            fieldName = "database_id";

```

Comment:
We already have the "id" on the document because of the "idField" ..  we should skip additionally setting this "id" on the document...